### ISSUE TITLE
ops.py重的Instance Normalization是不是有个地方搞错了？

### ISSUE THREAD

**Issue by Xinxiang7** [vanhuyz/CycleGAN-TensorFlow#120]
在ops.py文件中的第230行，是不是应该是`normalized = (input-mean)/inv`
文中的具体代码如下：
```python
def _instance_norm(input):
  """ Instance Normalization
  """
  with tf.variable_scope("instance_norm"):
    depth = input.get_shape()[3]
    scale = _weights("scale", [depth], mean=1.0)
    offset = _biases("offset", [depth])
    mean, variance = tf.nn.moments(input, axes=[1,2], keep_dims=True)
    epsilon = 1e-5
    inv = tf.rsqrt(variance + epsilon)
    normalized = (input-mean)*inv
    return scale*normalized + offset
```

**Comment by beshining** [comment]
> 在ops.py文件中的第230行，是不是应该是`normalized = (input-mean)/inv`
> 文中的具体代码如下：
> 
> ```python
> def _instance_norm(input):
>   """ Instance Normalization
>   """
>   with tf.variable_scope("instance_norm"):
>     depth = input.get_shape()[3]
>     scale = _weights("scale", [depth], mean=1.0)
>     offset = _biases("offset", [depth])
>     mean, variance = tf.nn.moments(input, axes=[1,2], keep_dims=True)
>     epsilon = 1e-5
>     inv = tf.rsqrt(variance + epsilon)
>     normalized = (input-mean)*inv
>     return scale*normalized + offset
> ```
**tf.rsqrt** computes **reciprocal of square root** of x element-wise, i.e. 1/sqrt(x). So, the code is right.

**Comment by Xinxiang7** [comment]
@beshining Thanks for your replay.
